docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,152 @@
 
 // tag::whats-new[]
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+[discrete]
+== Introducing Elastic AI Assistant use cases for Observability
+
+preview:[] Elastic Observability is excited to announce the initial (technical preview) release of the Elastic AI Assistant for Observability. Key observability workflows have been enhanced with generative AI (GAI) to help improve troubleshooting processes and provide automated explanations for information that is not easily understood. This will lead to better root cause analysis and improved MTTx.
+
+Elastic AI Assistant for Observability is incorporating generative AI in the following user workflows:
+
+* **Elastic AI Assistant for log message:** Provides the ability to use generative AI to look up the meaning of the log message details and help you find related messages.
++
+video::https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/bltf93d3d7ebbad9b82/64bef299320b7e5eedca7e54/LogAssistant.mov[width=640]
+
+* **Elastic AI Assistant for APM errors:** Provides the ability to explain an error or stack trace in APM and suggest remediations.
++
+video::https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/blt21e5befafed14b34/64bef0c6320b7e631bca7e3b/APMErrorAssistant.mov[width=640]
+
+* **Elastic AI Assistant for log alerts:** Provides the ability to use machine learning (ML) to explain what caused the spike in log messages and to use that information with generative AI to provide potential root cause and remediation steps.
++
+video::https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/bltf11ea707bfac2fde/64bef2b0d85ca66ef576f2bd/LogThresholdAssistant.mov[width=640]
+
+* **Elastic AI Assistant for host processes:** Provides the ability to get details on a process and how to optimize the process for resources like CPU or memory.
++
+video::https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/blt6f28aca285b49413/64bef27c7b7f98524d9feadc/HostProcessAssistant.mov[width=640]
+
+* **Elastic AI Assistant for profiling:** Provides the ability to explain the most expensive processes/functions in your organization and suggest optimizations — the AI assistant can use generative AI to explain any function and provide details on how to optimize the function.
++
+video::https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/blt45dc9c45747a265c/64bef2c1ab018e62ae51db2c/ProfilingAssistant.mov[width=640]
+
+All of the Elastic AI Assistant use cases mentioned above will be provided in 8.9 as a technical preview with the exception of log alerts (due to ML licensing requirements).
+
+This feature is available by configuring the Elastic AI Assistant, selecting the model, and entering in your credentials. Initially we will be supporting Azure OpenAI and OpenAI.
+
+// Follow the details in our {observability-guide}/obs-ai-assistant.html#obs-ai-set-up[documentation]. Here is a quick overview.
```

Comment:
This link is blocked by https://github.com/elastic/observability-docs/pull/3058.

Reply:
```suggestion
Ready to get started? See the {observability-guide}/obs-ai-assistant.html[documentation]. Here's a quick overview of the process:
```